src/dmd/dsymbolsem.d

```diff
@@ -6627,6 +6627,39 @@ void aliasSemantic(AliasDeclaration ds, Scope* sc)
         if (sc2 != sc)
             sc2.pop();
 
+        if (e)  // Try to convert Expression to Dsymbol
```

Comment:
please merge with the identical condition and comment below.

Reply:
The comment was wrong,  I fixed it.